packages/babel-plugin/src/styled/__tests__/behaviour.test.ts

```diff
@@ -269,20 +275,6 @@ describe('styled component behaviour', () => {
     expect(actual).toInclude('"--_1poneq5":ix((()=>{return props.color;})(),"\\"","\\"")');
   });
 
-  it('should move suffix and prefix of a dynamic arrow function with a body into an IIFE by preventing passing down invalid html attributes to the node', () => {
```

Comment:
Removed some tests in this PR as they were already being tested else where or have now become redundant due to the changes.